interactions/buttons/games.py

```diff
@@ -1178,6 +1182,23 @@ def number_to_emoji(self):
             GameString += "".join(row) + "\n"
         return GameString
 
+    def lost(self):
+        if self.has_empty:
+            return
+
+        board = [[b for b in i] for i in self.board]
```

Comment:
**Unnecessary looping.**
```py
>>> board
[[0, 1, 2, 3], [0, 1, 2, 3], [0, 1, 2, 3], [0, 1, 2, 3]]
>>> [[b for b in i] for i in board]
[[0, 1, 2, 3], [0, 1, 2, 3], [0, 1, 2, 3], [0, 1, 2, 3]]
>>>
```
Directly assign `board = self.board`

Reply:
Cuz changing the self.board updates board var